TradeItIosTicketSDK2/TradeItStepperInputTableViewCell.swift

```diff
@@ -31,7 +31,7 @@ class TradeItStepperInputTableViewCell: UITableViewCell {
         }
     }
 
-    func dismissKeyboard() {
+    @objc func dismissKeyboard() {
```

Comment:
Why do we need `@objc` here ?

Reply:
Functions passed to `#selector` need to be available to Obj-C. In Swift 3 it was inferred, but Swift 4 they are removing that inference.